lib/presenters/pull_requests_by_team.rb

```diff
@@ -2,24 +2,30 @@ module Presenters
   class PullRequestsByTeam
     def execute(teams:, ungrouped_pull_requests:)
       pull_requests_by_team = ungrouped_pull_requests.group_by { |pr| team_for_application(teams, pr.fetch(:application_name)) }
-      pull_requests_by_team.map do |team, pull_requests|
-        pull_requests_by_application = pull_requests.group_by { |pr| pr.fetch(:application_name) }
-
+      applications_by_team = pull_requests_by_team.map do |team, pull_requests|
```

Comment:
Like this variable name.  This execute method feels a bit busy though.  Extracting ` pull_requests_by_team`  into a method would help I think.